src/getHumanAge.test.js

```diff
@@ -3,8 +3,50 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return zeros if getHumanAge(0, 0)`, () => {
+    expect(getHumanAge(0, 0))
+      .toStrictEqual([0, 0]);
+  });
+
+  test(`should return zeros if getHumanAge(14, 14)`, () => {
+    expect(getHumanAge(14, 14))
+      .toStrictEqual([0, 0]);
+  });
+
+  test(`should return units if getHumanAge(15, 15)`, () => {
+    expect(getHumanAge(15, 15))
+      .toStrictEqual([1, 1]);
+  });
+
+  test(`should return units if getHumanAge(23, 23)`, () => {
+    expect(getHumanAge(23, 23))
+      .toStrictEqual([1, 1]);
+  });
+
+  test(`should return dueces if getHumanAge(24, 24)`, () => {
```

Comment:
The test description is not clear. It says it should return 'dueces' but it's unclear what 'dueces' means in this context.